plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3select/TestS3SelectPushdown.java

```diff
@@ -276,17 +276,17 @@ public void testShouldEnableSplits()
         assertTrue(isSplittable(true, schema, inputFormat, new Path("s3://fakeBucket/fakeObject.csv")));
         // Pushdown disabled
         assertTrue(isSplittable(false, schema, inputFormat, new Path("s3://fakeBucket/fakeObject.csv")));
+        // JSON
+        Properties newSchema = new Properties();
```

Comment:
Nit: call it `jsonSchema` maybe?

Reply:
Sounds good, I'll address this with the future review.